app/routes/circulars/circulars.server.ts

```diff
@@ -107,13 +138,12 @@ export async function search({
 
 /** Get a circular by ID. */
 export async function get(circularId: number): Promise<Circular> {
-  if (isNaN(circularId)) throw new Response(null, { status: 404 })
   const db = await tables()
   const result = await db.circulars.get({
     circularId,
   })
   if (!result)
-    throw new Response(null, {
+    throw new Response('The requested circular does not exist', {
```

Comment:
Whoops, looks like this patch undid some unrelated changes. Possibly a merge artifact?

Reply:
yeah, not sure how this happened, I think my fork may have been out of date when I created this branch or something along those lines. Resolved in [adf6556](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1026/commits/adf6556f2ea022c03eea7237e9efd56f2d737254)

Reply:
with most recent commit (64a34b2), double checked that the formatDateEpoch function works inside of the search function to limit the range of circulars by date. Manually changed the definition of startTime and endTime constants so instead of referencing startDate and endDate, they were hard-coded strings in the YYYY-MM-DD format. With this, the search function successfully limited the circulars to the provided date range (prior to the commit, I reverted the hard coded date range back to the constants). The backend for the search function seems to be functional as of now.